### PR TITLE
Internal tab

### PR DESCRIPTION
**Pull request by yuriy-fix**

### CODE REVIEW

File: src/vaadin-custom-field-mixin.html [hunk 1]
```diff
@@ -138,5 +154,12 @@
         this.validate();
       }
     }
+
+    /**
+     * Fired on Tab keydown triggered from the internal inputs,
+     * meaning focus will not leave the inputs.
+     *
+     * @event internal-tabbing
```
Comment (tomivirkki) [inline src/vaadin-custom-field-mixin.html]: The event name doesn't sound right, compare "clicking", "focusing"

Comment (yuriy-fix) [reply src/vaadin-custom-field-mixin.html]: Renamed to `internal-tab`

File: test/custom-field-keyboard-test.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          tab(customField.inputs[i]);
+        }
+
+        expect(internalTabSpy.callCount).to.equal(3);
+      });
+
+      it('should not fire `internal-tab` event on Tab from the last input', () => {
+        const internalTabSpy = sinon.spy();
+        customField.addEventListener('internal-tab', internalTabSpy);
+        customField.inputs[4];
```
Comment (tomivirkki) [inline test/custom-field-keyboard-test.html]: This wouldn't fire an event in any case. same below

Comment (yuriy-fix) [reply test/custom-field-keyboard-test.html]: Fixed.

**Pull request merged**